modules/aws-emr-sgs/security-group-rules.tf

```diff
@@ -539,6 +539,30 @@ resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
   description              = "Region Server - EMR Dashboard"
 }
 
+//Region Server rule - Additional SG for EMR Core with TAMR CIDRS for accessing HBase Metrics port
+resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_cidrs" {
```

Comment:
Name is a duplicate of above. These resource names need to be unique.
https://github.com/Datatamer/terraform-aws-emr/pull/45/files#diff-31f8037b12d08f6f61a61459bde4dbda5aa8125ace0c1728a10c51c332d4ea2bR519

Reply:
Oops, good catch. Fixed.